README.md

```diff
@@ -1,20 +1,20 @@
-# Membrane Multimedia Framework: FDK-AAC Element
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane-element-fdk-aac.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane-element-fdk-aac)
+# Membrane AAC FDK plugin
```

Comment:
I think we should standardize this somewhere because in some repos we have `Membrane X Plugin` and in the other `Membrane Multimedia Framework: X plugin` (e.g. in rtp or ice)

Reply:
Yeah :+1: